src/main/resources/fabric.mod.json

```diff
@@ -51,8 +51,8 @@
   "depends": {
     "fabricloader": ">=0.14",
     "fabric-api": "*",
-    "sodium": ">=0.5.3",
-    "minecraft": ">=1.20.2"
+    "sodium": ">=0.5.6",
+    "minecraft": ">=1.20.3"
```

Comment:
Note that Sodium 0.5.6 will support 1.20.1 and 1.20.3/1.20.4
Whether you want to backport is up to you and probably requires more changes